pkg/utils/imagevector/types.go

```diff
@@ -14,6 +14,15 @@
 
 package imagevector
 
+const (
+	// X86 is a constant for the 'x86' architecture.
+	X86 = "x86"
+	// Arm64 is a constant for the 'arm64' architecture.
+	Arm64 = "arm64"
+	// WildcardArchitecture is a constant for multi architectures.
+	WildcardArchitecture = "*"
```

Comment:
K8s maintains the architecture in the Node status and as a Node label. It uses the `goruntime.GOARCH` call to determine this field:

https://github.com/kubernetes/kubernetes/blob/23144130a815117194afbf3f703024ed76d5dac1/pkg/kubelet/nodestatus/setters.go#L440-L446


```yaml
kind: Node
# ...
status:
  nodeInfo:
    architecture: amd64
    # ...
    operatingSystem: linux
    osImage: Garden Linux 576.9
```


I don't see `x86` being a valid value for `goruntime.GOARCH` - ref https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63#goarch-values. The rather use `amd64` and `amd64p32`.

I don't have much expertise in the topic but shouldn't we stick to what K8s uses and maintains as allowed values?